docs/generatedcode/python.md

```diff
@@ -6,3 +6,147 @@ type: markdown
 ---
 
 # Python Generated Code Reference
+
+## Introduction
+
+gRPC Python relies on the protocol buffers compiler (`protoc`) to generate
+code.  It uses a plugin to supplement the generated code by plain `protoc`
+with gRPC-specific code.  For a `.proto` service description containing
+gRPC services, the plain `protoc` generated code are synthesized in the
```

Comment:
If we use the word "generated" consistently throughout rather than alternating between "generated" and "synthesized", does the doc get better (for the consistency) or worse (because the word appears unattractively too much)?